App.js

```diff
@@ -12,9 +12,10 @@ export default function App() {
   return (
     <PaperProvider theme={CustomTheme}>
       <NavigationContainer>
-        <Stack.Navigator headerMode='none' initialRouteName='Home'>
+        <Stack.Navigator headerMode='none' initialRouteName='Details'>
           <Stack.Screen name='ApiTesting' component={ApiTesting} />
           <Stack.Screen name='Home' component={HomeTabs} />
+          <Stack.Screen name='Details' component={DetailsTabs} />
```

Comment:
Isn't that temporary change? It looks like.

Reply:
Yes, it is. For now launching the app results in showing Details Screen instead of Main Screen, because it is new feature. It will be changed after implementing navigation between these two screens. Probably the ApiTesting screen will be also removed from the Stack.

Reply:
I think's that redundant but that's okay